app/src/main/java/io/homeassistant/companion/android/sensors/LastUpdateManager.kt

```diff
@@ -61,15 +61,15 @@ class LastUpdateManager : SensorManager {
         )
         val sensorDao = AppDatabase.getInstance(context).sensorDao()
         val allSettings = sensorDao.getSettings(lastUpdate.id)
-        val addNewIntent = allSettings.firstOrNull { it.name == ADD_NEW_INTENT }?.value ?: "false"
+        val addNewIntent = allSettings.firstOrNull { it.name == SETTING_ADD_NEW_INTENT }?.value ?: "false"
         val intentSetting = allSettings.firstOrNull { it.name == "intent" + allSettings.size }?.value ?: ""

```

Comment:
`"intent"` is also the name of the setting we may want to translate :)

Reply:
I've been thinking about that, too. But if you translate it into e.g. German, it doesn't make any sense anymore because it's more of a technical term. In German at least, the English term is much better in my opinion.

Reply:
yea I wasnt sure if it would translate well since its more of a Android specific term here right?

Reply:
Yes, when i talk/write in german about an android specific "intent" , i really say/write "Intent"